packages/guides-restructured-text/src/RestructuredText/Span/SpanParser.php

```diff
@@ -327,12 +328,21 @@ private function parseInterpretedText(): string
                     }
 
                     $inText = true;
+                    break;
+                case SpanLexer::WHITESPACE:
+                    // textroles may not contain whitespace, we are not in a textrole but have found a common colon
+                    if ($inText) {
+                        $part .= $token->value;
+                    } else {
+                        $rollback = true;
```

Comment:
You can use `break 2` to break out of 2 levels. that will simplify the code a bit.